app/services/idv/actions/verify_document_action.rb

```diff
@@ -33,19 +33,31 @@ def enqueue_job
           verify_document_capture_session_uuid_key,
         )
         document_capture_session.requested_at = Time.zone.now
-        document_capture_session.store_proofing_pii_from_doc({})
+        document_capture_session.store_proofing_pii_from_doc({}) # generates a result_id
 
-        VendorDocumentVerificationJob.perform(
-          document_capture_session_result_id: document_capture_session.result_id,
-          encryption_key: params[:encryption_key],
-          front_image_iv: params[:front_image_iv],
-          back_image_iv: params[:back_image_iv],
-          selfie_image_iv: params[:selfie_image_iv],
-          front_image_url: params[:front_image_url],
-          back_image_url: params[:back_image_url],
-          selfie_image_url: params[:selfie_image_url],
-          liveness_checking_enabled: params[:liveness_checking_enabled],
+        callback_url = Rails.application.routes.url_helpers.document_proof_result_url(
+          result_id: document_capture_session.result_id,
         )
+
+        LambdaJobs::Runner.new(
+          job_class: Idv::Proofer.document_job_class,
+          args: {
+            encryption_key: params[:encryption_key],
+            front_image_iv: params[:front_image_iv],
+            back_image_iv: params[:back_image_iv],
+            selfie_image_iv: params[:selfie_image_iv],
+            front_image_url: params[:front_image_url],
+            back_image_url: params[:back_image_url],
+            selfie_image_url: params[:selfie_image_url],
+            liveness_checking_enabled: params[:liveness_checking_enabled],
+            callback_url: callback_url,
+            trace_id: amzn_trace_id,
+          }
+        ).run do |doc_auth_result|
+          document_capture_session.store_proofing_pii_from_doc(doc_auth_result)
```

Comment:
In trying to test this in a real browser, I notice that we're not receiving the actual (fake) PII with `doc_auth_result`. Should we be?

I think we may still need some revisions to [our logic to determine document status verification as "done"](https://github.com/18F/identity-idp/blob/97fe583ff97ba1a8e6c43b6bafda160b9b9cc6e4/app/services/idv/actions/verify_document_status_action.rb#L70-L74) since it seems the actual result is being stored as `proofing_job_result.pii[:document_result]`.

Reply:
Great catch, I think that means we need to be need to be grabbing `:document_result` in the capture block locally, the controller already grabs that key: https://github.com/18F/identity-idp/blob/master/app/controllers/lambda_callback/document_proof_result_controller.rb

and I think I need to update the controller to actually contain the correct PII